api/src/main/java/org/pmiops/workbench/access/AccessUtils.java

```diff
@@ -34,13 +37,25 @@ public static DbAccessTier getAccessTierByShortNameOrThrow(
           DbAccessModuleName.RT_COMPLIANCE_TRAINING,
           DbAccessModuleName.TWO_FACTOR_AUTH);
 
-  public static final List<DbAccessModuleName> REQUIRED_MODULES_FOR_CONTROLLED_TIER =
-      ImmutableList.of(
-          DbAccessModuleName.CT_COMPLIANCE_TRAINING,
-          DbAccessModuleName.DATA_USER_CODE_OF_CONDUCT,
-          DbAccessModuleName.PROFILE_CONFIRMATION,
-          DbAccessModuleName.PUBLICATION_CONFIRMATION,
-          DbAccessModuleName.RAS_LOGIN_GOV,
-          DbAccessModuleName.RT_COMPLIANCE_TRAINING,
-          DbAccessModuleName.TWO_FACTOR_AUTH);
+  public static Collection<DbAccessModuleName> getRequiredModulesForRegisteredTierAccess() {
```

Comment:
these 2 are intentionally identical.  Does that deserve a comment?

Reply:
We need a function for the CT version (at least the way I have done it here) so I made RT also functions, for consistency